### PR TITLE
Add september presentation

### PR DESCRIPTION
**Pull request by flashadvocate**

### CODE REVIEW

**Comment by flashadvocate** [comment]
You may want to squash. Let me know if you'd like me to take care of this on my end and resubmit.

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+.idea/
```
Comment (cmgmyr) [inline .gitignore]: You can probably remove this file, these should go in your global gitignore file 🙂

File: 2017/09.md [hunk 2]
```diff
@@ -0,0 +1,341 @@
...
+**Who's looking?**
+**What are you looking for?**
+
+---
+
+>Thank you to Atlantic BT for hosting and refreshments!
+
+---
+
+# Tonight @ TrianglePHP
```
Comment (cmgmyr) [inline 2017/09.md]: This schedule is going to be a little different - the presentation is actually going to be before the "What's New" Do you want to move this to the first slide?

**Review by cmgmyr — CHANGES_REQUESTED** [review]

**Comment by cmgmyr** [comment]
I can squash on the merge, so no need to worry about it - thanks!

**Comment by cmgmyr** [comment]
Maybe add a quick mention about https://hacktoberfest.digitalocean.com/ too?

File: 2017/09.md (hunk 2 above)
Comment (flashadvocate) [reply 2017/09.md]: If the latest in php prez is going last, I can take out this part entirely

**Review by cmgmyr — APPROVED** [review]

**Comment by cmgmyr** [comment]
Looks great!

**Pull request merged**